src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.scss

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
As mentioned, we can now remove the `.align-row` CSS class in line 42, since it's not used anywhere

Reply:
Sorry for omitting that in the previous commit. The recent commit should resolve this.